### PR TITLE
Add autosnapshot docs.

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: website/pages/api-docs/system/storage/raftautosnapshots.mdx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  This is an Enterprise-only feature.
+---
+
+## Create/update an automated snapshots config
+
+**This endpoint requires sudo capability.**
+
+This endpoint creates or updates a named configuration.  Each configuration
+has an interval controlling how often snapshots are taken, a destination to
+which to write the snapshots, as well as a retention policy governing when
```
Comment (raskchanky) [inline website/pages/api-docs/system/storage/raftautosnapshots.mdx:19]: The destination bit reads a little awkwardly to me. I might suggest something like: `a destination where the snapshots are written`

File: website/pages/api-docs/system/storage/raftautosnapshots.mdx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+- `aws_access_key_id` `(string)` - AWS access key ID.
+
+- `aws_secret_access_key` `(string)` - AWS secret access key.
+
+- `aws_session_token` `(string)` - AWS session token.
+
+- `aws_s3_endpoint` `(string)` - AWS endpoint.  This is typically only set when
+  using a non-AWS S3 implementation like Minio.
+
+- `aws_s3_disable_tls` `(string)` - Disable TLS for the S3 endpoint.  This
```
Comment (raskchanky) [inline website/pages/api-docs/system/storage/raftautosnapshots.mdx:73]: Should this be `boolean`?

**Review by raskchanky — APPROVED** [review]
Looks good, just a few comments

File: website/pages/api-docs/system/storage/raftautosnapshots.mdx (hunk 2 above)
Comment (ncabatoff) [reply website/pages/api-docs/system/storage/raftautosnapshots.mdx]: Yes good catch.

File: website/pages/api-docs/system/storage/raftautosnapshots.mdx (hunk 1 above)
Comment (ncabatoff) [reply website/pages/api-docs/system/storage/raftautosnapshots.mdx]: Agreed

**Pull request merged**